### ISSUE TITLE
CloudEvents update breaks build

### ISSUE THREAD

**Issue by sebgoa** [triggermesh/knative-lambda-sources#30]
```
$ go build .
# github.com/triggermesh/knative-lambda-sources/awssqs
./main.go:211:9: assignment mismatch: 2 variables but clients.CloudEvents.Send returns 3 values
```

**Comment by sebgoa** [comment]
See https://github.com/cloudevents/sdk-go/blob/master/cmd/samples/http/sender/main.go#L87

**Comment by a-edakin** [comment]
Issue was solved in #28 packages update
